src/sdk/pynni/tests/test_tuner.py

```diff
@@ -118,7 +118,7 @@ def search_space_test_all(self, tuner_factory, supported_types=None, ignore_type
             search_space_all = json.load(fp)
         if supported_types is None:
             supported_types = ["choice", "randint", "uniform", "quniform", "loguniform", "qloguniform",
-                               "normal", "qnormal", "lognormal", "qlognormal"]
+                               "normal", "qnormal", "lognormal", "qlognormal", "mutable"]
```

Comment:
since the `_type` is `mutable_layer`, why we are not consistent with the naming rule here?

Reply:
Because I used some tricks on deciding whether the testcase should be successful or not, which is based on "splitting with underscore".

Reply:
Nevertheless refactored to get rid of this trick.